src/testing/syntax/syntaxtest.sh

```diff
@@ -82,8 +82,8 @@ then
     echo "Using $threads threads"
 fi
 
-# scan each of the php files
-phpScanResults=$(find . -not \( -path ./vendor -prune \) -type f -name \*.php | xargs -L1 -P$threads -I {} sh -c "php --syntax-check {} || :")
+# scan each of the php files except testdata folder
+phpScanResults=$(find . -not \( -path ./vendor -path ./nomos/agent_tests/testdata \) -prune -type f -name \*.php | xargs -L1 -P$threads -I {} sh -c "php --syntax-check {} || :")
```

Comment:
I read about `find ` and applied same. I tested before commit and there was no error and it was because `find` wasn't returning anything as you point out but I couldn't catch that. Sorry about that:(